ckan/templates/home/snippets/about_text.html

```diff
@@ -9,12 +9,12 @@
 
 <p>CKAN is used by governments and user groups worldwide and powers a variety
 of official and community data portals including portals for local, national
-and international government, such as the UK’s <a href="http://data.gov.uk">data.gov.uk</a> and the
-European Union’s <a href="http://publicdata.eu/">publicdata.eu</a>, the Brazilian <a href="http://dados.gov.br/">dados.gov.br</a>, Dutch and
+and international government, such as the UK’s <a href="https://data.gov.uk">data.gov.uk</a> and the
+European Union’s <a href="https://data.europa.eu/">data.europa.eu</a>, the Brazilian <a href="https://dados.gov.br/">dados.gov.br</a>, Dutch and
```

Comment:
The EU portal is not powered by CKAN 

```suggestion
United States <a href="https://catalog.data.gov/">catalog.data.gov</a>, the Brazilian <a href="https://dados.gov.br/">dados.gov.br</a>, Dutch and
```